src/login.html

```diff
@@ -4,14 +4,21 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles/common.css" />
+    <style>
+      .error{
+        color: red;
+      } 
+    </style>
     <title>Document</title>
   </head>
   <body>
-      <form id="login-form">
-        <input id="user" type="user" placeholder="User" required/>
+      <form id="loginForm">
+        <input id="user" type="user" placeholder="User" />
+        <div id="errorUser" class="error"></div>
         <br>
```

Comment:
remove all these br and hr, use display: flex with gap to add space between elements